src/components/submission/feed-entry.vue

```diff
@@ -71,7 +74,10 @@ export default {
         : 'edited';
     },
     comment() {
-      return this.entry.notes != null ? this.entry.notes : this.entry.body;
+      const comment = this.entry.notes != null ? this.entry.notes : this.entry.body;
+      return comment != null
+        ? DOMPurify.sanitize(marked(comment, { gfm: true, breaks: true }))
```

Comment:
Is there a way to add attributes to links? We usually specify `target="_blank"` for links that leave Central. And maybe it'd be good to specify `rel="noopener"` or something along those lines.